database/bigquery/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas==1.0.4
```

Comment:
Is pandas a "transient" dependency. Required by bigquery? It isn't being referenced in the code.

Reply:
Used this during testing and forgot to remove it from the requiements.txt. Removed it!